packages/python/accern_data/accern_data.py

```diff
@@ -365,6 +365,9 @@ def __init__(
         self._first_error = True
         self._expected_records: List[int] = []
 
+    def reset(self) -> None:
+        self._expected_records = []
```

Comment:
why do we store this in the object in the first place? it's only local info for the current download so it should just be a variable in the function

Reply:
right!